Content.Server/Advertise/MessagePackPrototype.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Robust.Shared.Prototypes;
+
+namespace Content.Server.Advertise;
+
+[Serializable, Prototype("messagePack"), Virtual]
+public partial class MessagePackPrototype : IPrototype
```

Comment:
if this is only virtual for backwards compatibility with `AdvertisementsPackPrototype` i'd say just remove the other one and make the breaking changes. you're a big boy.

Reply:
Yeah, my concern was for downstreams that might have added vending machines. But yeah, it won't be a difficult change for them to make.